### ISSUE TITLE
Does this work with quantised models?

### ISSUE THREAD

**Issue by aaa3334** [mrousavy/react-native-fast-tflite#93]
Hi!

I have tried a few different models with this but when I try one that is quantised, I get that the model refuses to load.. Just wondering if it only works with non-quantised models or if it would also work with quantised?

Thanks in advance!

**Comment by aaa3334** [comment]
I have tried about 4 different quantised models now and I just get: 
 LOG  MODEL not loaded

export const textRecognition = () => {
  const tf = useTensorflowModel(
    require("../../assets/models/ocrattempt.tflite")
  );
  const { resize } = useResizePlugin();

  const recognizeText = useCallback(
    (frame: Frame) => {
      "worklet";
      if (tf.state !== "loaded" || tf.model === undefined) {
        console.log("MODEL not loaded");
        return null;
      }

Unsure how to collect the logs (i am not really seeing any in xcode but can look more closely).

Models tried are the smaller selfie segmenter by google and this ocr one: https://www.kaggle.com/models/spsayakpaul/east-text-detector/TfLite/dr/1

**Comment by francesco-clementi-92** [comment]
I'm having the same problem

**Comment by lucksp** [comment]
yes, similar issue - on iOS with GPU enabled, on any model from Vertex after September 2024 (approx date).

Are you seeing this on iOS, Android or both?
Are you enabling GPU?
What dates were you models trained?
What version of this library are you using?
